src/slick.grid.ts

```diff
@@ -4605,6 +4661,10 @@ export class SlickGrid<TData = any, C extends Column<TData> = Column<TData>, O e
       while (Utils.isDefined(columnIdx = cacheEntry.cellRenderQueue.pop())) {
         node = x.lastChild as HTMLElement;
 
+        //no idea why node would be null here but apparently it is..
+        if (!node && this._options.nonce) {
```

Comment:
This i'm unsure of why node would be null here, but it being null doesn't seem to affect the UI

Reply:
it might be null if it's the last row in the grid then there's no next child (or `lastChild`) I would assume so it makes sense to keep a check against node

Reply:
will change the check to just be the if(!node) then